.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: node_js
 node_js:
+- '4.1'
```

Comment:
Travis doesn't support it yet: https://docs.travis-ci.com/user/languages/javascript-with-nodejs